src/components/Team.tsx

```diff
@@ -6,18 +6,18 @@ export default function TeamSection() {
   const [activeImage, setActiveImage] = useState("");
   return (
     <section id="team" className="bg-black">
-      <div className="h-[1530px] md:h-[1000px] w-10/12 m-auto md:w-full flex flex-col md:grid grid-cols-12 pt-10">
-        <h2 className="uppercase text-white font-tandem-mono-medium text-xs md:mb-0 mb-10 col-start-3 col-span-1">
+      <div className="m-auto flex  w-10/12 grid-cols-12 flex-col py-28 md:grid  md:w-full">
+        <h2 className="col-span-1 col-start-3 mb-10 font-tandem-mono-medium text-xs uppercase text-white md:mb-0">
           ■ Team
         </h2>
-        <div className="col-start-5 col-span-8">
-          <p className="text-4xl font-tandem-condensed-medium uppercase text-gray-400 md:mb-20">
+        <div className="col-span-8 col-start-5">
+          <p className="font-tandem-condensed-medium text-4xl uppercase text-gray-400 md:mb-20">
             <span className="text-white">Our dedicated team specialises</span>{" "}
             in creating impactful digital solutions for arts, music, and
             community focused projects.
           </p>
-          <div className="col-start-5 col-span-8">
-            <div className="flex md:flex-row flex-col md:mt-0 mt-20 gap-10 items-end font-tandem-mono uppercase text-white h-[550px]">
+          <div className="col-span-8 col-start-5">
+            <div className="font-tandem-mono mt-20 flex  flex-col items-end gap-10 uppercase text-white md:mt-0 md:flex-row">
```

Comment:
```suggestion
            <div className="font-tandem-mono mt-20 flex  flex-col items-end gap-10 uppercase text-white md:mt-0 md:flex-row h-[550px]">
```
Believe this fixed height was required to stop the images jumping up and down when resizing, perhaps some of the other tailwind classes you removed were part of this fix too